aip/0192.md

```diff
@@ -91,6 +91,14 @@ Comments **may** "link" to another component (service, method, message, field,
 enum, or enum value) by using the fully-qualified name of the element as a
 Markdown reference link. For example: `[Book][google.example.v1.Book]`
 
+### External links
+
+Comments **may** link to external pages to provide background information
+beyond what is described in the public comments themselves. External links
+**must** use absolute (rather than relative) URLs, and should not assume the
```

Comment:
Minor suggestion:

```suggestion
**must** use absolute (rather than relative) URLs, including the protocol (usually `https`), and should not assume the
```

(Note: Run prettier after accepting for proper line breaks.)